tortoisebeacon/tortoise_beacon.go

```diff
@@ -901,8 +873,8 @@ func (tb *TortoiseBeacon) getSignedProposal(epoch types.EpochID) ([]byte, error)
 	signature := tb.vrfSigner.Sign(p)
 	tb.Log.With().Debug("Calculated signature",
 		log.Uint64("epoch_id", uint64(epoch)),
-		log.String("proposal", util.Bytes2Hex(p)),
-		log.String("signature", util.Bytes2Hex(signature)))
+		log.String("proposal", string(p)),
```

Comment:
i think you still want to use hex in logs (or log.Binary). string([]byte{..}) won't be readable most of the time

Reply:
You're right, I replaced too much stuff